### PR TITLE
Add basic internal notifications

### PR DESCRIPTION
**Pull request by pociej**
For now only for table invitations

### CODE REVIEW

File: userApp/imports/api/InternalNotification/publish.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Tables } from '/imports/api/Tables';
+import { NOTIFICATION_TYPE__TABLE_INVITATION } from './'
+
+Meteor.publish('internalNotifications', function (roomId) {
+  //NOTE : we should consider using events emmiter for consitency here instead of 
+  //collection observers 
+
+  //FOR now only new tables invitation 
+  let initializing = true;
+  const handle = Tables.find().observeChanges({
```
Comment (zdraganov) [inline userApp/imports/api/InternalNotification/publish.js:10]: Is this synchronous?

File: userApp/imports/api/Tables/methods.js [hunk 2]
```diff
@@ -32,7 +33,12 @@ export const createNewTable = new ValidatedMethod({
         return { position, username, userId: user._id, atTable: false };
       });
 
+      console.log("ownwer", owner);
```
Comment (zdraganov) [inline userApp/imports/api/Tables/methods.js:36]: You can remove this.

File: userApp/imports/api/InternalNotification/publish.js (hunk 1 above)
Comment (pociej) [reply userApp/imports/api/InternalNotification/publish.js:10]: no its not why ?

Comment (pociej) [reply userApp/imports/api/InternalNotification/publish.js:10]: the purpose of initializing flag is not to sent invitation for existing tables, just for new one

**Pull request merged**